crates/bitwarden-ipc/src/ipc_client.rs

```diff
@@ -292,19 +292,15 @@ where
         cancellation_token: Option<CancellationToken>,
     ) -> Result<Request::Response, RequestError<Crypto::SendError>>
     where
-        Request: RpcRequest + TryInto<Vec<u8>> + TryFrom<Vec<u8>>,
-        Request::Response: TryInto<Vec<u8>> + TryFrom<Vec<u8>>,
-        <Request as TryInto<Vec<u8>>>::Error: std::fmt::Display,
-        <Request::Response as TryFrom<Vec<u8>>>::Error: std::fmt::Display,
+        Request: RpcRequest,
     {
         let request_id = uuid::Uuid::new_v4().to_string();
-        let mut response_subscription: IpcClientTypedSubscription<RpcResponseMessage> =
-            self.subscribe_typed().await?;
+        let mut response_subscription: IpcClientTypedSubscription<
+            IncomingRpcResponseMessage<Request::Response>,
+        > = self.subscribe_typed().await?;
```

Comment:
Non blocking nit, but if you move the type to the function, you can let the compiler infer some of the types:
```suggestion
        let mut response_subscription = self
            .subscribe_typed::<IncomingRpcResponseMessage<_>>()
            .await?;
```

Reply:
Nice, fixed!